pdc/apps/utils/utils.py

```diff
@@ -59,3 +61,43 @@ def read_permission_for_all():
 
 def get_model_name_from_obj_or_cls(obj_or_cls):
     return ContentType.objects.get_for_model(obj_or_cls).model
+
+
+class RelatedNestedOrderingFilter(OrderingFilter):
+    """
+    Extends OrderingFilter to support ordering by fields in related models
+    using Django orm '__'.
+
+    This class is copied and changed from the patch of the github issue.
+    https://github.com/encode/django-rest-framework/issues/1005
```

Comment:
Just check, we don't need add the "reverse relation", the instance is not 'ForeignObjectRel' or 'OneToOneRel'.

And I have already changed the url to the comment which includes the patch,  good suggestion!